Examples/microcontroller/STM32F302R6T6TR.json

```diff
@@ -0,0 +1,11407 @@
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "maxValue": 1.3,
+            "conditions": [
+              "IIO = +20 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "maxValue": 0.4,
+            "conditions": [
+              "IO = +6 mA",
+              "2 V < VDD < 2.7 V"
+            ]
+          }
+        ]
+      },
+      "voh": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -0.4,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "CMOS port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "minValue": 2.4,
+            "conditions": [
+              "TTL port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -1.3,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "IIO = +20 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -0.4,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "IO = +6 mA",
+              "2 V < VDD < 2.7 V"
+            ]
+          }
+        ]
+      },
+      "absVmax": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": 0.4,
+            "relativeValueOperator": "add"
+          }
+        ]
+      },
+      "absVmin": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VSS",
+            "relativeValueModifier": -0.3,
+            "relativeValueOperator": "add"
+          }
+        ]
+      },
+      "vmax": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "maxValue": 5.5
+          }
+        ]
+      },
+      "imax": {
+        "values": [
+          {
+            "siUnit": "milliamp",
+            "maxValue": 25
+          }
+        ]
+      },
+      "inputLeakage": {
+        "values": [
+          {
+            "siUnit": "microamp",
+            "maxValue": 0.1,
+            "conditions": [
+              "digital mode",
+              "VSS ≤ VIN ≤ VDD"
+            ]
+          },
+          {
+            "siUnit": "microamp",
+            "maxValue": 10,
+            "conditions": [
+              "VDD ≤ VIN ≤ 5 V"
+            ]
+          }
+        ]
+      },
+      "esd": true
+    },
```

Comment:
there is an option for an internal pull-up. 

I didn't list them initially because they're not enabled by default and I was worried this would be confusing. I submitted an issue to discuss: https://github.com/edatasheets/digital-datasheets/issues/98